commit

```diff
@@ -19,5 +19,5 @@ $TODO_FULL_SH archive
 cd "$TODO_DIR"
 git diff
 if [ "$mode" != "-v" ]; then
-	git commit -a $mode --allow-empty-message -m ""
+	git commit -q -a $mode -m "$HOST: $(date -Iseconds)"
```

Comment:
I'm currently on an OS X box, where `$HOST` is unset - however, `$HOSTNAME` does work (as does the `hostname` command`); would that work for you as well?

As for `-q`, I kinda like having the commit summary as success feedback - so perhaps that could be passed in via `mode`? That might be overkill though and you might be better off forking this tiny file.